management/server/geolocation/geolocation_test.go

```diff
@@ -16,23 +14,17 @@ import (
 
 // from https://github.com/maxmind/MaxMind-DB/blob/main/test-data/GeoLite2-City-Test.mmdb
 var (
-	mmdbPath     = "../testdata/GeoLite2-City-Test.mmdb"
-	mmdbFilename = "GeoLite2-City.mmdb"
+	mmdbPath     = "../testdata/GeoLite2-City_20240305.mmdb"
+	mmdbFilename = "../testdata/geonames_20240305.db"
```

Comment:
in the test, the contents of `mmdbPath` is copied to the path of `path.Join(tempDir, mmdbFilename)`, but I think with this change, its copied from `"../testdata/GeoLite2-City_20240305.mmdb"` to `{tempDir}/geonames_20240305.db` when it should be `"{tempDir}/GeoLite2-City_20240305.mmdb"`